OmsAgent/omsagent.py

```diff
@@ -46,100 +49,279 @@
 if os.path.exists(ext_log_path):
     os.chmod(ext_log_path, 700)
 
+
 def main():
     waagent.LoggerInit('/var/log/waagent.log','/dev/stdout', True)
     waagent.Log("%s started to handle." %(ExtensionShortName))
 
+    # Determine the operation being executed
+    operation = None
     try:
         for a in sys.argv[1:]:
```

Comment:
Whether an error gets thrown about NoneType, or operation being undefined, and error will get thrown if operation doesn't get defined in the checks below. In my opinion, a NoneType error would be more explicit.